### PR TITLE
pep8 changes in skl2onnx and add flake8 check to ci build

### PR DESCRIPTION
**Pull request by prabhat00155**

### CODE REVIEW

File: skl2onnx/_supported_operators.py [hunk 1]
```diff
@@ -28,18 +28,18 @@
 from sklearn.multiclass import OneVsRestClassifier
 
 # Tree-based models
+from sklearn.ensemble import ExtraTreesClassifier
```
Comment (wenbingl) [inline skl2onnx/_supported_operators.py]: import as 'alphabetic order' is recommended in PEP8?

File: skl2onnx/__init__.py [hunk 2]
```diff
@@ -15,9 +15,9 @@
 __model_version__ = 0
 
 
-from .convert import convert_sklearn
-from ._supported_operators import update_registered_converter
-from ._parse import update_registered_parser
+from .convert import convert_sklearn # noqa
```
Comment (wenbingl) [inline skl2onnx/__init__.py]: it looks a very normal import, why # noqa?

Comment (prabhat00155) [reply skl2onnx/__init__.py]: This import is not directly getting used in this file, so flake8 throws a warning.

File: skl2onnx/_supported_operators.py (hunk 1 above)
Comment (prabhat00155) [reply skl2onnx/_supported_operators.py]: It's not mentioned in PEP8, but it improves readability.

Comment (prabhat00155) [reply skl2onnx/_supported_operators.py]: It's not mentioned in PEP8, but it improves readability.

**Pull request merged**